server/commands/write.go

```diff
@@ -65,6 +65,13 @@ func (c *WriteCommand) validateTuplesets(ctx context.Context, req *openfgapb.Wri
 		}
 	}
 
+	for _, tk := range deletes {
+		// For delete, we only need to ensure it is well form but no need to validate whether relation exists
+		if err := tupleUtils.ValidateUser(tk); err != nil {
```

Comment:
We should be validating the whole tuple. We should just simplify this to:

```
for _, tk := range deletes {
    err := tupleUtils.ValidateTupleKey(tk)
    if err != nil {
        // handle error
    }
}
```

because this only validates the user component.

Reply:
There is no ValidateTupleKey - most of the validation already happened via the gRPC framework.  Only the user needs to be validated separately.  As for ValidateTuple, it validates User + the relations (which in the case of delete, we don't need to).